### ISSUE TITLE
Port 15100 might be occupied

### ISSUE THREAD

**Issue by sid2021** [ko4life-net/ko#198]
#### Description
I'm trying to run the project locally using provided instructions but after compiling (`All.sln`) and running the code I'm getting:
```
KnightOnLine: Failed to connect to game server: [10061].
Port 15100 might be occupied.
```
I checked if the port `15100` is occupied in my OS (Windows 11) `netstat -ano | findstr :15100` but it seems it's not the case. 

Could you please provide some tips on how to tacle this issue?

Just for you information I made a local setup using:
Visual Studio 2022 Community
SQL Server 2022 
Microsoft ODBC Driver 18 for SQL Server

#### Screenshots
![image](https://github.com/ko4life-net/ko/assets/62474990/61c32b9a-5dce-4e64-b5ae-6be4d166a453)

**Comment by stevewgr** [comment]
@sid2021, yes, the error message is a bit misleading. I remember I had this issue and was trying to debug this once. I found out that the Windows API call for whatever reason was failing to create the socket. Probably system related or permissions, not sure, but as funny as it sounds, restarting my computer helped solving this issue. Let us know how you solved this, since it would be nice fixing this issue at its core.
